cappy.py

```diff
@@ -123,11 +137,12 @@ def handle_inbound_request(self, message):
             request_id = message['id']
             args = message['args']
         except KeyError as e:
+            print(e)
```

Comment:
We could just `raise` but than it would make more sense to remove the `as e` altogether and use
```python
except KeyError:
    raise
```